moesifasgi/logger_helper.py

```diff
@@ -113,50 +117,62 @@ def get_user_id(self, middleware_settings, request, response, request_headers, d
         return user_id
 
     @classmethod
-    def get_company_id(cls, middleware_settings, request, response, debug):
+    async def get_company_id(cls, middleware_settings, request, response, debug):
         company_id = None
         try:
             identify_company = middleware_settings.get('IDENTIFY_COMPANY', None)
             if identify_company is not None:
-                company_id = identify_company(request, response)
+                if inspect.iscoroutinefunction(identify_company):
```

Comment:
since we're using this check `inspect.iscoroutinefunction(function)` at many palces, does it make sense to create a helper function which checks if it's a coroutine or not?

Reply:
Added